templates/partials/comment.rs.html

```diff
@@ -3,14 +3,14 @@
 @use plume_models::users::User;
 @use routes::*;
 
-@(ctx: BaseContext, comm: &Comment, author: User, in_reply_to: Option<&str>)
+@(ctx: BaseContext, comm: &Comment, author: User, in_reply_to: Option<&str>, blog: &str, slug: &str)
 
 <div class="comment u-comment h-cite" id="comment-@comm.id">
     <a class="author u-author h-card" href="@uri!(user::details: name = author.get_fqn(ctx.0))">
         @avatar(ctx.0, &author, Size::Small, true, ctx.1)
         <span class="display-name p-name">@author.name(ctx.0)</span>
         <small>@author.get_fqn(ctx.0)</small>
-    </a>
+	    </a>
```

Comment:
why this indent?

Reply:
There is a 100% chance my vim is still fucking up the indent in html templates, I have no idea why but in those rs.html it absolutely want to add tabs everywhere